### DISCUSSION TITLE
Flashes when app window is being resized

### DISCUSSION THREAD

**Discussion by dmi-dmi-dmi** [tauri-apps/tauri#2970] [Q&A] [answered]
I have created a new tauri project with `yarn create tauri-app` where I picked `create-react-app` as UI recipe.

Affer launching dev environment right after I noticed grey flashes when app window is being resized.
My initial thought was to build release version which should work faster and, hence, have no such flashes.

However, flashes are still present in the release version. Pleasee see attached video
https://user-images.githubusercontent.com/19564404/143600115-46b3d435-32a4-4cc6-90a7-b267e7ea86d6.mp4

Am I doing something wrong? Is it possible to supress the flashes?

`yarn tauri info` output
```
> yarn tauri info
yarn run v1.22.11
$ tauri info
 app:spawn Running "/home/dmi-dmi-dmi/tmp/tauri-test/node_modules/@tauri-apps/cli/bin/tauri-cli tauri info" +0ms


Operating System - Arch Linux, version Unknown X64

Node.js environment
  Node.js - 16.7.0
  @tauri-apps/cli - 1.0.0-beta.10
  @tauri-apps/api - 1.0.0-beta.8

Global packages
  npm - 7.20.3
  yarn - 1.22.11

Rust environment
  rustc - 1.56.1
  cargo - 1.56.0

App directory structure
/src-tauri
/public
/build
/src
/node_modules
/.git

App
  tauri.rs - 1.0.0-beta.8
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../build
  devPath - http://localhost:3000
  framework - React

Done in 9.00s.
```

**Comment by FabianLars** [comment, marked as answer]
That's webkit being not so perfect (again).
For example on windows it can resize without flashing, but still kinda laggy, the same as in any normal browser.
I don't think there is much we can improve here on our side tho 🤔

**Reply by dmi-dmi-dmi** [reply]
Ah, that is so unfortunate. I am thinking about ways to mitigate this. Can I turn off resizing on tauri app window? If no if is possible to adjust background color so at least flash doesn't involve different (lighter) color?

Thanks a lot for prompt response

**Reply by FabianLars** [reply]
> Can I turn off resizing on tauri app window?

Yes. <https://tauri.studio/en/docs/api/config#tauri.windows.resizable>. The js/rust window apis provide a similar function.

> adjust background color

Not yet, but there are open issues/requests for that. But i'm not 100% sure if that would fix it, could be just the renderer being stupid, but i hope not lol

**Comment by Elendiar** [comment]
I encountered the same problem when displaying a popover window and changing its size before displaying. The part of the window by which the window is enlarged blinks white. 
I solved this by setting the size after hiding not with width: 0 height: 0, but on the contrary, a larger size ( like width: 1000 height: 1000 if i know my popover always be lesser) than the window will be, then it cuts the size without blinking

**Reply by alexiokay** [reply]
could u show us your code? 

EDIT: I chagned transparent to true in tauri config, so now its fine.
